people_manager.rb

```diff
@@ -0,0 +1,79 @@
+require './student'
+require './teacher'
+
+class PeopleManager
+  attr_reader :list_people
+
+  def initialize
+    @list_people = []
+  end
+
+  # People Block
+  def insert_student
+    print 'Insert Name:'
+    name = gets.chomp
+    print 'Insert Age:'
+    age = gets.chomp
+    print 'Has permission? [y, n] '
+    permission = gets.chomp
+    permission = permission == 'y'
+    @list_people.push(Student.new(age: age, name: name, parent_permission: permission))
+    puts 'Student created Succesfully'
+  end
```

Comment:
- [ ] I think we can improve better on the Single Responsibility Principle here. Right now this method is in charge of both gathering the inputs from the user and creating the student. 

I think it would be better if it only creates the student. For that, we can gather all inputs from the user on a different class explicitly created for that or you can do it on the main.rb file.